WeKan v8.43: Cannot find module '@meteorjs/reify/lib/runtime'

### Bug Description

# Server Setup Information
WeKan version: v8.43 (was 8.42 before)
Did you configure root-url correctly so Wekan cards open correctly? Yes.
Operating System: Linux 3.10.0-1160.139.1.el7.tuxcare.els3.x86_64
Deployment Method: bundle
Http frontend if any: Nginx
Meteor-Version: (2.16 in v8.42)
Node.js Version: 14.21.4
MongoDB Version: 5.0.32
Webbrowser: Safari 26.0.1 (19622.1.22.118.4)

# Problem description
After upgrading from 8.42 → 8.43 and starting the service, I get a 502 … Please refer to the supervisord.log content below …

Wondering how to fix this? Could you please help, @xet7 ?
Thank you, BF

### Steps to Reproduce

Start wekan service …

### Relevant Logs

```shell
2026-03-28 09:55:25,834 DEBG 'wekan' stderr output:
internal/modules/cjs/loader.js:936
  throw err;
  ^

Error: Cannot find module '@meteorjs/reify/lib/runtime'
Require stack:
- /home/USER/wekan/wekan-8.43/bundle/programs/server/runtime.js
- /home/USER/wekan/wekan-8.43/bundle/main.js
    at FunctUSER.Module._resolveFilename (internal/modules/cjs/loader.js:933:15)
    at FunctUSER.Module._load (internal/modules/cjs/loader.js:776:27)
    at Module.require (internal/modules/cjs/loader.js:1005:19)
    at require (internal/modules/cjs/helpers.js:107:18)
    at enable (/home/USER/wekan/wekan-8.43/bundle/programs/server/runtime.js:34:5)
    at Object.<anonymous> (/home/USER/wekan/wekan-8.43/bundle/main.js:4:40)
    at Module._compile (internal/modules/cjs/loader.js:1116:14)
    at Object.Module._extensUSERs..js (internal/modules/cjs/loader.js:1145:10)
    at Module.load (internal/modules/cjs/loader.js:981:32)
    at FunctUSER.Module._load (internal/modules/cjs/loader.js:821:12) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/home/USER/wekan/wekan-8.43/bundle/programs/server/runtime.js',
    '/home/USER/wekan/wekan-8.43/bundle/main.js'
  ]
}
```

### Additional Context

_No response_

--- a/Dockerfile
+++ b/Dockerfile
@@ -204,6 +204,7 @@ cd /home/wekan/app
 wget "https://github.com/wekan/wekan/releases/download/v${VERSION}/wekan-${VERSION}-${WEKAN_ARCH}.zip"
 unzip "wekan-${VERSION}-${WEKAN_ARCH}.zip"
 rm "wekan-${VERSION}-${WEKAN_ARCH}.zip"
+npm install --production --prefix ./bundle/programs/server
 mv /home/wekan/app/bundle /build
 
 # Restore original tar

